cmd/commands/ceph.go

```diff
@@ -31,7 +31,7 @@ var CephCmd = &cobra.Command{
 	DisableFlagParsing: true,
 	Args:               cobra.MinimumNArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
-		context := GetContext()
-		fmt.Println(exec.RunCommandInOperatorPod(context, cmd.Use, args, OperatorNamespace, CephClusterNamespace, true))
+		clientsets := GetClientsets()
+		fmt.Println(exec.RunCommandInOperatorPod(cmd.Context(), clientsets, cmd.Use, args, OperatorNamespace, CephClusterNamespace, true))
```

Comment:
One of the core reasons to make this change, versus letting it stand, is to make sure all commands explicitly use `cmd.Context()` for their context.

Reply:
> Documentation in context.Context states never to embed a context.Context in a struct. It should always be passed to functions and methods, ideally as the first parameter.

We don't need to pass the context as a pointer? Seems like it would be similar behavior to pass it by value as to pass it around with a struct, but I didn't go dig into the whole issue.

Reply:
context.Context is an interface, and I think it fails to compile if a `*` is in front of the param.